src/main/java/com/questhelper/quests/princealirescue/PrinceAliRescue.java

```diff
@@ -169,10 +169,10 @@ public void setupConditions()
 	{
 		inCell = new ZoneRequirement(cell);
 		hasWigPasteAndKey = new Conditions(dyedWig.alsoCheckBank(questBank), paste.alsoCheckBank(questBank), key.alsoCheckBank(questBank));
-		givenKeyMould = new Conditions(true, LogicType.OR,
+		givenKeyMould = new Conditions(true, LogicType.OR,	// TODO quest journal widget text outdated
 			new WidgetTextRequirement(119, 3, true, "I have duplicated a key, I need to get it from"),
 			new WidgetTextRequirement(119, 3, true, "I got a duplicated cell door key"),
-			new WidgetTextRequirement(WidgetInfo.DIALOG_NPC_TEXT, "Pick the key up from Leela."),
+			new WidgetTextRequirement(WidgetInfo.DIALOG_NPC_TEXT, "I'll send it to Leela once it's ready."),
```

Comment:
This is untested, but should trigger the condition if the player does not close out of the NPC dialogue before this message is displayed.